### ISSUE TITLE
Update ITL and Docs for memory-windows - show used

### ISSUE THREAD

**Issue by teclogi** [Icinga/icinga2#6640]
The Documentation and the ITL should be updated for the option -U (--show-used) in the command "memory-windows".

## Expected Behavior

The ITL for "memory-windows" should be changed to:

```
object CheckCommand "memory-windows" {
        command = [ PluginDir + "/check_memory.exe" ]

        arguments = {
                "-w" = {
                        value = "$memory_win_warn$"
                        description = "Warning Threshold"
                }
                "-c" = {
                        value = "$memory_win_crit$"
                        description = "Critical Threshold"
                }
                "-u" = {
                        value = "$memory_win_unit$"
                        description = "Use this unit to display memory"
                }
                "-U" = {
                        set_if = "$memory_win_show_used$"
                        description = "Show used memory instead of the free memory"
                }
        }

        //The default
        vars.memory_win_unit = "mb"
        vars.memory_win_warn = "10%"
        vars.memory_win_crit = "5%"
}
```

The documentation should also be adapted accordingly.

## Current Behavior

The ITL for "memory-windows" looks currently like this:

```
object CheckCommand "memory-windows" {
        command = [ PluginDir + "/check_memory.exe" ]

        arguments = {
                "-w" = {
                        value = "$memory_win_warn$"
                        description = "Warning Threshold"
                }
                "-c" = {
                        value = "$memory_win_crit$"
                        description = "Critical Threshold"
                }
                "-u" = {
                        value = "$memory_win_unit$"
                        description = "Use this unit to display memory"
                }
        }

        //The default
        vars.memory_win_unit = "mb"
        vars.memory_win_warn = "10%"
        vars.memory_win_crit = "5%"
}
```

## Possible Solution

Update ITL and documentation.

## Context

I want to display the used memory of a system.
Graphs can be displayed better by this methods.

## Your Environment

* Version used (`icinga2 --version`): 2.9.1
* Operating System and version: Debian 9
* Enabled features (`icinga2 feature list`):
* Icinga Web 2 version and modules (System - About): 2.6.1

**Comment by mcktr** [comment]
Good catch, thanks :+1: 

Would you mind to create a Pull Request? :)

**Comment by mcktr** [comment]
This is fixed in #6646. Thanks again @teclogi :+1:
